packages/react/src/hooks/accounts/useSigner.ts

```diff
@@ -7,19 +7,25 @@ import {
 } from '@wagmi/core'
 import * as React from 'react'
 
-import { QueryConfig } from '../../types'
+import { QueryConfig, QueryFunctionArgs } from '../../types'
 import { useQuery } from '../utils'
 
 export type UseSignerConfig = Omit<
   QueryConfig<FetchSignerResult, Error>,
   'cacheTime' | 'staleTime' | 'enabled'
->
+> & {
+  chainId?: number
+}
```

Comment:
Should this use `FetchSignerArgs` instead?

Reply:
Good catch.